MotMystere.cpp

```diff
@@ -2,19 +2,53 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
 
 int main()
 {
-    std::cout << "Hello World!\n";
+   
+    
+    //1 : On demande de saisir un mot
+
+    string motMystere("Procra");
+    cout << "Entrer un mot : ";
+    std::cin >> motMystere;
```

Comment:
At the beginning of the file, you declared "using namespace std". This means that instead of having to type 
> std::someFunction()
you can directly type
> someFunction()

If you look carefully, you will see that this is what you are doing with cout, you don't type std::cout but just cout.
So same goes with cin, you can just type cin instead of std::cin

Note that it relies on the fact that you declared 'using namespace std'
If you did not add that line at the beginning, you would have to use 'std::' everywhere it is normally needed.

Reply:
got it!